### PR TITLE
Kogito 5336 update docs consoles

### PR DESCRIPTION
**Pull request by AjayJagan**
Updating examples in accordance with SSO integration in the consoles.

This PR can be merged after kiegroup/kogito-apps#914 has been merged.

It contains the following changs:

update Readme in accordance with the new SSO integration.
update docker-compose.yml to integrate keycloak server and managment and task console props
update the bpmn diagram to use proper actors

### CODE REVIEW

File: process-usertasks-springboot-with-console/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -75,11 +92,13 @@ services:
...
         condition: service_started
+      keycloak:
+        condition: service_healthy
     volumes:
       - ./svg/:/home/kogito/data/svg/
     environment:
-      KOGITO_DATAINDEX_HTTP_URL: http://${DOCKER_GATEWAY_HOST}:8180
-      KOGITO_MANAGEMENT_CONSOLE_PROPS: -Dkogito.svg.folder.path=/home/kogito/data/svg
+      KOGITO_DATAINDEX_HTTP_URL: http://${DOCKER_GATEWAY_HOST}:8180/graphql
+      KOGITO_MANAGEMENT_CONSOLE_PROPS: -Dkogito.consoles.keycloak.config.url=http://${DOCKER_GATEWAY_HOST}:8480/auth -Dkogito.consoles.keycloak.config.health-check-url=http://${DOCKER_GATEWAY_HOST}:8480/auth/realms/kogito/.well-known/openid-configuration -Dkogito.svg.folder.path=/home/kogito/data/svg
```
Comment (bxf12315) [inline process-usertasks-springboot-with-console/docker-compose/docker-compose.yml:101]: You use different keyclock's config from process-usertasks-with-security-oidc-quarkus-with-console.
common config is easy to maintain.

File: process-usertasks-quarkus-with-console/README.md [hunk 2]
```diff
@@ -176,20 +190,20 @@ Once the service is up and running you can make use of the **Hiring** applicatio
 ```json
 {   
     "candidate": {
-        "name": "Jon Snow",
-        "email": "jsnow@example.com",
+        "name": "jdoe",
```
Comment (pefernan) [inline process-usertasks-quarkus-with-console/README.md]: Well, this could stay as it was since it was random data :D

Comment (pefernan) [inline process-usertasks-quarkus-with-console/README.md:193]: This change wasn't required but we can keep it if you want.

**Review by pefernan — CHANGES_REQUESTED** [review]

**Comment by pefernan** [comment]
@AjayJagan I would do only one last change and it is updating the README files inside of the docker-compose folder to specify that keycloak will be started too. That should only be in the examples without security. Apart from that everything is fine. The examples work perfect with the new consoles. Congratulations!

File: process-usertasks-quarkus-with-console/README.md (hunk 2 above)
Comment (AjayJagan) [reply process-usertasks-quarkus-with-console/README.md]: hehe changed them back

**Review by pefernan — APPROVED** [review]
+1 thanks @AjayJagan

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by cristianonicolai** [comment]
@AjayJagan @pefernan seems like the url to connect with the data index needs to contain `/graphql`  now. If we really need to change that, we should also update the docs, add to the release notes so we can inform users about this change as this is likely to break when people migrate from a previous version. Can you guys follow up on that, please?

**Comment by AjayJagan** [comment]
> @AjayJagan @pefernan seems like the url to connect with the data index needs to contain `/graphql` now. If we really need to change that, we should also update the docs, add to the release notes so we can inform users about this change as this is likely to break when people migrate from a previous version. Can you guys follow up on that, please?

@cristianonicolai  - sure will follow up on this

**Pull request merged**